src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.targets

```diff
@@ -304,7 +304,7 @@ Copyright (c) .NET Foundation. All rights reserved.
       </PropertyGroup>
     </When>
     
-    <When Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework' and '$(OutputType)' == 'Exe'">
+    <When Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework' and '$(IsExecutable)' == 'true'">
```

Comment:
@eerhardt, I would think this would be valid and part of setting `IsExecutable=true` would involve having uses configure the F5 information (to use xunit, nunit, dotnet, etc...)

However, maybe having a separate variable to disable (or explicitly enable) F5 for some scenarios would be desirable